File: app/src/main/java/org/bedu/okayapp/Trivia/Estudio.java
package org.bedu.okayapp.Trivia;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;
import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.views.YouTubePlayerView;
import org.bedu.okayapp.R;
import org.bedu.okayapp.Temas.ShowTemas;

public class Estudio extends AppCompatActivity {
    private YouTubePlayerView youTubePlayerView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_estudio);
        getSupportActionBar().hide();

        youTubePlayerView = findViewById(R.id.youtubePlayer);
        getLifecycle().addObserver(youTubePlayerView);

        Button botoncito = (Button) findViewById(R.id.boton_regreso);
        botoncito.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                startActivity(new Intent(Estudio.this,ShowTemas.class));
            }
        });
    }
}

File: app/src/main/java/org/bedu/okayapp/Trivia/Estudio.kt
package org.bedu.okayapp.Trivia
//ketzalli
//muestra el video de youtube *se debe corregir
import android.os.Bundle
import android.widget.Button
import android.widget.Toast
import com.google.android.youtube.player.YouTubeBaseActivity
import com.google.android.youtube.player.YouTubeInitializationResult
import com.google.android.youtube.player.YouTubePlayer
import com.google.android.youtube.player.YouTubePlayerView
import org.bedu.okayapp.R
import java.util.regex.Pattern

class Estudio : YouTubeBaseActivity() {

    val API_KEY = "AIzaSyD6J7ssBgAaKyEORp7VceghPcm4u8Nh6ew"
    val videoPrueba="bgo9dJB_icw"


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_estudio)
        var youtubePlayer = findViewById<YouTubePlayerView>(R.id.youtubePlayer)
        val reproducir = findViewById<Button>(R.id.reproducir)
        var youtubePlayerInit:YouTubePlayer.OnInitializedListener
        youtubePlayerInit=object:YouTubePlayer.OnInitializedListener{
            override fun onInitializationSuccess(
                p0: YouTubePlayer.Provider?,
                p1: YouTubePlayer?,
                p2: Boolean
            ) {
                p1?.loadVideo(videoPrueba)
            }

            override fun onInitializationFailure(
                p0: YouTubePlayer.Provider?,
                p1: YouTubeInitializationResult?
            ) {
                Toast.makeText(applicationContext,"Errorrrrrrr",Toast.LENGTH_SHORT).show()
            }

        }
        reproducir.setOnClickListener{
            youtubePlayer.initialize(API_KEY,youtubePlayerInit)
        }
    }
}